next-i18next.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     locales: ['en', 'zh', 'ko'],
     // TODO: This is because there is currently no full i18n translation support,
     // so switching to other languages is temporarily disabled.
-    localeDetection: false,
+    // localeDetection: false,
```

Comment:
Also blocked by the alert, updated by https://github.com/Magickbase/nervos-official-website/pull/108/commits/98b47e9e64e5c0459666612c4490d0f8e32ebdb8

Reply:
The following TODO and localeDetection can also be removed

Reply:
Ah I recalled why `zh` is still disabled. The translation is done by gpt-3.5 and requires review from the foundation

Reply:
`zh` and `locale detection` were disabled again by https://github.com/Magickbase/nervos-official-website/pull/108/commits/82b28ac74b0e37a03e6bdf6e53abccc42d70037d because the content should be reviewed.

These updates could be merged first so community can work on the internationalization.